ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -35,17 +35,17 @@ cp "${SHARED_DIR}/install-config.yaml" "${dir}/"
 mkdir -p ~/.ssh
 cp "${SSH_PRIV_KEY_PATH}" ~/.ssh/
 
-TF_LOG=debug openshift-install --dir="${dir}" create cluster 2>&1 | grep --line-buffered -v password &
+TF_LOG=debug openshift-install --dir="${dir}" create cluster 2>/dev/null
```

Comment:
Wouldn't this mean that all the error messages that people usually see in prow job ui will no longer be available??

Reply:
> Wouldn't this mean that all the error messages that people usually see in prow job ui will no longer be available??

They can `tail -f .openshift_install.log`, but yeah, the couldn't just look at standard streams anymore.  Did you want to keep the `grep` here and double up on the uploaded assets?

Reply:
See the failure https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/logs/release-openshift-origin-installer-e2e-gcp-4.5/918#1:build-log.txt%3A45

I think most of the people seeing CI job failures expect to see these upfront ... And not having these show is not great!

Reply:
ah, good point about the build log.  Reverted this hunk with 1a7b314f9 -> 068c5de4c.